sla/sla-core/sla-personalization/pom.xml

```diff
@@ -53,6 +59,11 @@
 			<groupId>com.sun.jersey</groupId>
 			<artifactId>jersey-client</artifactId>
 		</dependency>
+
+		<dependency>
+			<groupId>it.polimi.modaclouds.qos-models</groupId>
+			<artifactId>qos-models</artifactId>
+		</dependency>
```

Comment:
Of course. Just paste here the needed dependency. When both PRs have been accepted, we can submit one with the dependency in the root.